src/custom_auth/views.py

```diff
@@ -1,51 +1,17 @@
-from django.contrib.auth import login
```

Comment:
Wow, that's a lot of deletion. 😲

Reply:
Some redundant codes need to be removed ;)